scss/bitstyles/tools/_font-size.scss

```diff
@@ -0,0 +1,42 @@
+@use 'sass:map';
+@use '../settings/typography';
+@use 'unit-conversions';
+
+// Palette
+//
+// **Returns requested font-size from the global `typography.$font-sizes` map.**
+// e.g. `font-size: font-size.get('h1');`
+//
+// @param {String} $font-size  The font-size, as named in the `typography.$font-sizes` map
+//
+// @param {String} $breakpoint   Optional. Defaults to `base`, you can request the font-size at other breakpoints as defined in the `typography.$font-sizes` map
+//
+
+@function get($font-size, $breakpoint: 'base') {
+  @if map.has-key(typography.$font-sizes, $breakpoint) {
+    $font-sizes: map.get(typography.$font-sizes, $breakpoint);
+    $base-size: typography.$font-size-base-small;
+
+    @if map.has-key($font-sizes, 'html') {
+      $base-size: map.get($font-sizes, 'html');
+    }
+
+    @if map.has-key($font-sizes, $font-size) {
+      @if $font-size == 'html' {
+        @return unit-conversions.px-to-rem(map.get($font-sizes, $font-size), 16px);
```

Comment:
Where do the hardcoded 16px come from?

Reply:
That’s the standard browser default font-size. I could make it a variable, so the name would make that clearer